Validation/RecoTrack/python/plotting/trackingPlots.py

```diff
@@ -732,6 +735,8 @@ def _formatOrNone(num, func):
     def headers(self):
         return [
             "Efficiency",
+            "Number of TrackingParticles (after cuts)",
+            "Number of matched TrackingParticles",
```

Comment:
@rovere The efficiency denominator and numerator are added to the summary tables here (as you asked some time ago).